oe_theme.libraries.yml

```diff
@@ -36,18 +25,6 @@ context_nav:
   dependencies:
     - core/drupal
     - oe_theme/base
-file_translations:
```

Comment:
Library was removed but is still being loaded on file_translation.ui_patterns.yml

Reply:
Very good catch.